tests/ci/build_run_benchmarks.sh

```diff
@@ -65,77 +62,55 @@ prod_pid=$!
 taskset -c 7 ./aws-lc-prod/fips_build/tool/awslc_bm -timeout 3 -json > aws-lc-prod_fips_bm.json &
 prod_fips_pid=$!
 
-taskset -c 8 ./"${PR_FOLDER_NAME}"/build/tool/ossl_bm -timeout 3 -json > ossl_bm.json &
-ossl_pid=$!
-taskset -c 9 ./"${PR_FOLDER_NAME}"/build/tool/bssl_bm -timeout 3 -json > bssl_bm.json &
-bssl_pid=$!
 
 # wait for benchmarks to finish
 wait "${pr_pid}"
 wait "${pr_fips_pid}"
 wait "${prod_pid}"
 wait "${prod_fips_pid}"
-wait "${ossl_pid}"
-wait "${bssl_pid}"
 
 # once all those are done, we want to run the trusttoken benchmarks sequentially
 # (running them in parallel caused severe bias problems)
 taskset -c 4 ./"${PR_FOLDER_NAME}"/build/tool/awslc_bm -filter trusttoken -timeout 3 -json > aws-lc-pr_tt_bm.json
 taskset -c 5 ./"${PR_FOLDER_NAME}"/fips_build/tool/awslc_bm -filter trusttoken -timeout 3 -json > aws-lc-pr_tt_fips_bm.json
 taskset -c 6 ./aws-lc-prod/build/tool/awslc_bm -filter trusttoken -timeout 3 -json > aws-lc-prod_tt_bm.json
 taskset -c 7 ./aws-lc-prod/fips_build/tool/awslc_bm -filter trusttoken -timeout 3 -json > aws-lc-prod_tt_fips_bm.json
-taskset -c 8 ./"${PR_FOLDER_NAME}"/build/tool/ossl_bm -filter trusttoken -timeout 3 -json > ossl_tt_bm.json
-taskset -c 9 ./"${PR_FOLDER_NAME}"/build/tool/bssl_bm -filter trusttoken -timeout 3 -json > bssl_tt_bm.json
 
 # convert results from .json to .csv
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-pr_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-pr_fips_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-prod_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-prod_fips_bm.json
-python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py ossl_bm.json
-python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py bssl_bm.json
 
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-pr_tt_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-pr_tt_fips_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-prod_tt_bm.json
 python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py aws-lc-prod_tt_fips_bm.json
-python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py ossl_tt_bm.json
-python3 "${PR_FOLDER_NAME}"/tests/ci/benchmark_framework/convert_json_to_csv.py bssl_tt_bm.json
 
 # once we have csvs, we want to update the main benchmark results files with the sequential trusttoken results
```

Comment:
do we need these trust token numbers at all? I feel like we never cared about them and it's just generating noise

Reply:
I don't know that we need them but I don't think it'll hurt to have and I doubt it'll cause too much noise since they should be easy enough to ignore